# @ref adasis_v2.h
uint8 type

uint16 offset_from_start

uint16 cyclic_offset

uint16 cyclic_segment_num

int16 offset_from_ego

float32 value
